scipy/stats/_morestats.py

```diff
@@ -3197,23 +3197,18 @@ def fligner(*samples, center='median', proportiontocut=0.05):
     nonparametric tests by extensive simulations and they conclude that the
     tests proposed by Fligner and Killeen (1976) and Levene (1960) appear to be
     superior in terms of robustness of departures from normality and power
-    [3]_.
+    [1]_.
```

Comment:
This should actually remain [3] in the updated references.  The sentence is referring to a conclusion from the paper by Conover et al.